governance.md

```diff
@@ -323,3 +323,19 @@ Existing **Tier 2** Partners:
 
 -   Bloomberg (Jason Grout, Sylvain Corlay)
 -   Google (Kester Tong, Kayur Patel, Mark Sandler)
+
+
+Changing the Governance
```

Comment:
Maybe "Changing the"

* "Governance Model"
* "Governance Policy"
* "Governance Document"

?